### DISCUSSION TITLE
Bootstrap theme not displaying correctly

### DISCUSSION THREAD

**Discussion by Andream98** [vueform/vueform#250] [Questions] [answered]
Hello, I'm trying out the library but I can't manage to get the Bootstrap theme working. The form does not get displayed properly

Here is a [StackBlitz with a basic example](https://stackblitz.com/edit/github-vbdsxe?file=src%2Fstyle.scss,vueform.config.js,src%2Fmain.js,src%2FApp.vue)

**Comment by adamberecz** [comment, marked as answer]
When using form steps inline, you always have to wrap things in the `#empty` slot (otherwise everything will be rendered as form elements):
https://stackblitz.com/edit/github-vbdsxe-r6ares?file=src%2FApp.vue

```vue
<Vueform>
  <template #empty>
    <FormSteps>
      <!-- ... -->
    </FormSteps>
    <FormElements>
      <!-- ... -->
    </FormElements>
    <FormStepsControls />
  </template>
</Vueform>
```

**Reply by Andream98** [reply]
Oh ok, I didn't notice this detail in the docs! Thanks!
